instance/tests/integration/integration_instance.py

```diff
@@ -42,6 +43,9 @@
 
 # Tests #######################################################################
 
+@override_settings(
+    DEFAULT_LMS_PREVIEW_DOMAIN_PREFIX='integ-',
```

Comment:
This confused me at first too - the LMS preview domain is what's being used to determine the name of nova instance name by default. I couldn't figure out a better way to override it while I was working to try and get the instance names changed.

This isn't strictly necessary for my change (it filters keypair rather than instance name), but the instance names looked similar to: `edxapp-preview-7ea2d`
This is fine, but there are a bunch of other VMs in there beginning with `edxapp-preview-` and name clashing is possible.